test/spec/controllers/solrSettings.js

```diff
@@ -129,18 +129,18 @@ describe('solrSettingsCtrl', function() {
         scope.searchSettings.publishSearcher();
 
         expect(scope.searchSettings.fieldSpecStr).toEqual('catch_line text');
-        expect(scope.searchSettings.searchArgsStr).toEqual('q=choice of law\n&defType=edismax\n&qf=catch_line text\n&pf=catch_line');
+        expect(scope.searchSettings.searchArgsStr).toEqual('q=choice of law&defType=edismax&qf=catch_line text&pf=catch_line');
```

Comment:
Yep, I think one just for showing the other is stored for showing in the URL input, although `searchArgsStr ` could be converted from `searchArgsStrShow `if I'm not remembering wrong there is a lot of logic around `searchArgsStr` being set, any recommendation on this?